CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+### pg_auto_failover v1.6.1 (July 7, 2021) ###
+
+This release contains monitor schema changes, so the monitor extension gets
+a version bump from 1.5 to 1.6, and this is the first release in the 1.6
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
+
+#### Added
+* Feature crash recovery before pg rewind (#656)
+* Allow building pg_auto_failover with Postgres 14. (#716)
+* Fix Postgres 14 beta2 incompatibility. (#739)
+* Track nodes current timeline on the monitor. (#730)
```

Comment:
```suggestion
* Track nodes current timeline on the monitor and use it during election. (#730)
```

Reply:
Normally we would use the wording of the git commit (or PR merge), though in that case it's better to help the users read the notes, so added too.